dist/milo_ui.bundle.js

```diff
@@ -528,19 +528,35 @@ function dispatchInputMessage() {
 }
 
 
-function toISO8601Format(date) {
-    var dateArr = [
-        date.getFullYear(),
-        pad(date.getMonth() + 1),
-        pad(date.getDate())
-    ];
-
-    var dateStr = dateArr.join('-');
+function toISO8601Format(date, utc) {
+    var dateStr = [
+        get('FullYear'),
+        pad(get('Month') + 1), // JS Date API indexes month from 0
+        pad(get('Date'))
+    ].join('-');
 
     return dateStr;
 
     function pad(n) { return n < 10 ? '0' + n : n; }
+    function get(field) { return date['get' + (utc ? 'UTC' : '') + field ](); }
```

Comment:
Util function to avoid the following:

`utc ? date.getUTCFullYear() : date.getFullYear()`
`utc ? date.getUTCMonth) : date.getFullYear()`
`utc ? date.getUTCFullYear() : date.getFullYear()`